config/manifests/bases/orchestrator-operator.clusterserviceversion.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
       managed resources (operands). To prevent data loss or disruption, you'll need
       to manually delete them.
     operators.openshift.io/valid-subscription: '["Red Hat Developer Hub"]'
-    repository: https://github.com/parodos-dev/orchestrator-helm-operator
+    repository: https://github.com/rhdhorchestrator/orchestrator-helm-operator
```

Comment:
I would avoid making any updates not related to the spec changes. Besides, this is being covered in @pkliczewski 's renaming PR (#427 )

Reply:
I had added this initially when making changes to this file and forgot to remove it after reviewing Piotr's PR. Will make the change.

Reply:
Done